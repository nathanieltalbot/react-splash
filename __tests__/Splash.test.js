/*
 * React Splash Test Suite
 *
 * Copyright Â© Roman Nosov 2016
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */

import React from 'react';
import { mount } from 'enzyme';
import Splash from '../src/Splash';


describe('Splash', () => {
  it('renders a initial view', () => {
    const content = mount(
      <Splash />
    );
    expect(content.html()).toMatchSnapshot();
  });
});
