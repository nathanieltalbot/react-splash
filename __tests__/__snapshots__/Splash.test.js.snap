exports[`Splash renders a initial view 1`] = `"<div class=\"splash\"><img src=\"https://source.unsplash.com/weekly\"><h1><span class=\"splash-word\"><span class=\"splash-left\">R</span><span class=\"splash-left\">e</span><span class=\"splash-left\">a</span><span class=\"splash-left\">c</span><span class=\"splash-left\">t</span></span><span class=\"splash-word\"><span class=\"splash-right\">S</span><span class=\"splash-right\">p</span><span class=\"splash-right\">l</span><span class=\"splash-right\">a</span><span class=\"splash-right\">s</span><span class=\"splash-right\">h</span></span></h1></div>"`;
